exonum/src/blockchain/config.rs

```diff
@@ -46,6 +46,9 @@ pub struct StoredConfiguration {
     pub validator_keys: Vec<ValidatorKeys>,
     /// Consensus algorithm parameters.
     pub consensus: ConsensusConfig,
+    /// Number of votes to commit configuration.
```

Comment:
/// Number of votes to commit configuration. -->    /// Number of votes required to commit new configuration.